a3p-integration/proposals/z:acceptance/vaults.test.js

```diff
@@ -115,133 +95,327 @@ test.serial('remove IST', async t => {
+});
 
-  t.is(
-    atomBalanceAfter,
-    atomBalanceBefore + vaultCollateral,
-    'The ATOM balance should increase by the vault collateral amount',
-  );
-  t.is(vaultState, 'closed', 'The vault should be in the "closed" state.');
+test.serial(
+  'Minting Fee is applied to users debt when creating a vault and minting more IST',
+  async t => {
+    const mint = 5n;
+    const collateral = mint * 2n;
+    await openVault(GOV1ADDR, mint.toString(), collateral.toString());
+
+    const { adjustedToMintAmount } = await calculateMintFee(
+      mint,
+      VAULT_MANAGER,
+    );
+    t.log('mint + fee:', adjustedToMintAmount.value);
+
+    const { vaultID, debt: debtAfterOpenVault } =
+      await getLastVaultFromAddress(GOV1ADDR);
+    t.log('vault debt after open:', debtAfterOpenVault);
+
+    t.is(
+      debtAfterOpenVault,
+      adjustedToMintAmount.value,
+      'The vault Debt should be equal to mint + fee',
+    );
+
+    await adjustVault(GOV1ADDR, vaultID, { wantMinted: 1.0 });
+
+    const { adjustedToMintAmount: adjustedToMintAmountAfter } =
+      await calculateMintFee(1n, VAULT_MANAGER);
+    t.log('wantMinted + fee:', adjustedToMintAmountAfter.value);
+
+    const { debt: debtAfterAdjustVault } =
+      await getLastVaultFromAddress(GOV1ADDR);
+    t.log('vault debt after adjust:', debtAfterAdjustVault);
+
+    t.is(
+      debtAfterAdjustVault,
+      debtAfterOpenVault + adjustedToMintAmountAfter.value,
+      'The vault Debt after adjusting should be equal to debt after open + wantMinted + fee',
+    );
+  },
+);
+
+test.serial('confirm that Oracle prices are being received', async t => {
+  /*
+   * The Oracle for ATOM and stATOM brands are being registered in the offer made at file:
+   * a3p-integration/proposals/n:upgrade-next/verifyPushedPrice.js
+   * which is being executed during the use phase of upgrade-next proposal
+   */
+  const ATOMManagerIndex = 0;
+  const BRANDS = ['ATOM'];
+  const BASE_ID = 'n-upgrade';
+  const oraclesByBrand = generateOracleMap(BASE_ID, BRANDS);
+
+  const latestRoundId = await getPriceFeedRoundId(BRANDS[0]);
+  const roundId = latestRoundId + 1;
+
+  await verifyPushedPrice(oraclesByBrand, BRANDS[0], 10, roundId);
+
+  const atomQuote = await getPriceQuote(BRANDS[0]);
+  t.log('price quote:', atomQuote);
+  t.is(atomQuote, '+10000000');
+
+  const vaultQuote = await getVaultPrices(ATOMManagerIndex);
+  t.log('vault quote:', vaultQuote.value[0].amountOut.value);
+
+  t.true(vaultQuote.value[0].amountIn.brand.includes(' ATOM '));
+  t.is(vaultQuote.value[0].amountOut.value, atomQuote);
 });
+
+test.serial(
+  'Confirm that vaults that existed before the most recent upgrade continue to be useable',
+  async t => {
+    /*
+     * The long-living-vault user is being created and used to open a vault in the n:upgrade-next proposal USE phase.
+     * The offer to open a vault is implemented in a3p-integration/proposals/n:upgrade-next/openVault.js
+     */
```

Comment:
This is acceptable on an interim basis, but it would better and cleaner to add a vault in the use phase of something in `agoric-3-proposals` so we don't have to rely on retaining  this open vault in `n:upgrade-next`, which will continuously evolve as we push out releases. 

After this PR, would you make a PR that opens a vault there that we can use the next time we need one?